alchemy-web/docs/providers/cloudflare/openauth.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Cloudflare OpenAuth
+description: Learn how to create OAuth authentication workers using OpenAuth and Hono on Cloudflare Workers with Alchemy.
+---
+
+# OpenAuth
+
+A [Cloudflare Worker](https://developers.cloudflare.com/workers/) that serves an [OpenAuth](https://openauth.js.org/) Hono application for OAuth authentication with multiple providers.
+
+OpenAuth provides a secure, edge-compatible authentication solution with support for GitHub, Google, Discord, Facebook, and other OAuth providers. It integrates seamlessly with Cloudflare KV for session storage and Hono for API routing.
+
+## Minimal Example
+
+Create a basic OpenAuth worker with GitHub authentication:
+
+```ts
+import { OpenAuth, KVNamespace, secret } from "alchemy/cloudflare";
+
+const authStore = await KVNamespace("auth-sessions", {
+  title: "auth-sessions"
+});
+
+const auth = await OpenAuth("auth", import.meta, {
+  providers: {
+    github: {
+      clientId: secret.env.GITHUB_CLIENT_ID,
+      clientSecret: secret.env.GITHUB_CLIENT_SECRET,
```

Comment:
Should be `alchemy.secret(process.env.GITHUB_CLIENT_ID)` etc. consistently across the api